src/authservice/chart/templates/secret.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ include "authservice.fullname" . }}
+  name: {{ include "authservice.fullname" . }}-static
   namespace: {{ .Release.Namespace }}
   labels:
     {{- include "authservice.labels" . | nindent 4 }}
```

Comment:
I think the Helm-based configuration's (defaults at least) should _exactly_ match what the operator code would generates?

Please see my other comments on the Authservice operator code and make sure the global config is in sync. This actually looks _more_ configurable (good, assuming the defaults match).

I assume the reason for this duplication is to allow people a way to inject some custom config into the authservice config? The code duplication seems not great. But I guess that use case makes sense somewhat

Reply:
Yeah - need to do some cleanup here. This secret is not currently being used so we will likely be deleting the template.

The only things we need to follow up on here is:
- ensuring we have a way to get the CA cert to pepr (this could be as simple as another env passed to pepr for the config, we may need this anyways for some other CA related issues)
- deciding if we want to support non-pepr managed authservice chains (I think short-term answer is no)